config/database.py

```diff
@@ -1,11 +1,11 @@
 """ Database Settings """
-
 import os
+import logging
```

Comment:
I just cleaned and re-organized the imports...

Reply:
Do we need this file? I thought the database.py file is inside `tests/integrations/config/databse.py`?

Reply:
in Masonite yes...but in standalone ORM here we need this file or a file defined by DB_CONFIG_PATH

Reply:
So do we just need to set the env in the GitHub actions file?